Mathlib/Topology/UniformSpace/CompactConvergence.lean

```diff
@@ -398,11 +398,10 @@ theorem tendsto_of_tendstoLocallyUniformly (h : TendstoLocallyUniformly (fun i a
 implies locally uniform convergence.
 
 See also `ContinuousMap.tendsto_iff_tendstoLocallyUniformly`, especially for T2 spaces. -/
```

Comment:
I can't select it, but could you change `If every point has a compact neighbourhood` to `In a weakly locally compact space` in this docstring?